README.md

```diff
@@ -1 +1,73 @@
-# Rails_Budget_App
\ No newline at end of file
+# Rails_Budget_App
+# ![](https://img.shields.io/badge/Microverse-blueviolet) 
+
+## [Live Version]
```

Comment:
- [ ] Nice job @principles31 , I can not see the live link of your project, kindly include that

Reply:
@VICTOR-CHUKWU Thank you for the feedback

Reply:
Hey, the link you provided seems to be broken. Kindly add a valid link so the next reviewer can check the deployed version of your application. 
![image](https://user-images.githubusercontent.com/33268927/191803483-73fba625-d340-4099-a5ca-7c38bc03ead1.png)